tmt/steps/execute/run.sh

```diff
@@ -87,6 +87,19 @@ tmt_main () {
     local IFS="$IFS_b"
 }
 
+# Store test result into the 'results.yaml' file
+# Parameters: test result log
+tmt_store_result() {
+    local test="$1"
+    local result="$2"
+    local log="$3"
+
+    local results="$tmt_WD/execute/results.yaml"
```

Comment:
In no way you're ensuring the folder exists / file is writable. Writing to that file may result in an unhandled error, which would abort the execution. Would you mind if I'd write it consistently with, with chesk, the rest of the files?

Reply:
Results handling functionality moved to the `simple` execute plugin.